tests/io/v0_0_0/test_http_backend.py

```diff
@@ -132,48 +135,6 @@ def test_numpy(self):
 
         self.assertTrue(np.array_equal(list(result.tiles())[0].numpy_array, tile.numpy_array))
 
-    def test_numpy_requires_seekable(self):
```

Comment:
In your PR description, you should explain why this was removed (basically what you explained to me in person is reasonable).